packages/marketplace/src/resources/snippets.ts

```diff
@@ -179,4 +179,10 @@ export default [
     "code": "section.main-shelf-shelf[aria-label='Recently played'] { visibility: hidden; position: absolute; }",
     "preview": "resources/assets/snippets/remove-recently-played.png",
   },
+  {
+    "title": "Modernize custom app icons",
+    "description": "Gives icons from the sticky list a thicker border",
```

Comment:
thicker != more modern

```suggestion
    "title": "Make custom app icons more thick",
    "description": "Gives icons from the sticky list a thicker border",
```

such, edit preview file name as well

Reply:
yeah that's a much better name. I'll also change the file name in the next commit